store/src/main/java/com/nytimes/android/external/store3/base/impl/MemoryPolicy.java

```diff
@@ -24,12 +24,14 @@
     private final long expireAfterAccess;
     private final TimeUnit expireAfterTimeUnit;
     private final long maxSize;
+    private final long maximumInFlightRequestsDurationInSeconds;
 
-    MemoryPolicy(long expireAfterWrite, long expireAfterAccess, TimeUnit expireAfterTimeUnit, long maxSize) {
+    MemoryPolicy(long expireAfterWrite, long expireAfterAccess, TimeUnit expireAfterTimeUnit, long maxSize, long maximumInFlightRequestsDurationInSeconds) {
         this.expireAfterWrite = expireAfterWrite;
         this.expireAfterAccess = expireAfterAccess;
         this.expireAfterTimeUnit = expireAfterTimeUnit;
         this.maxSize = maxSize;
+        this.maximumInFlightRequestsDurationInSeconds = maximumInFlightRequestsDurationInSeconds;
```

Comment:
I think allowing a timeunit + duration would be more consistent with rest of api